tests/e2e/specs/shopper/cart-checkout/account.test.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../../utils';
+import { SIMPLE_PHYSICAL_PRODUCT_NAME } from '.../../../../utils/constants';
+
+/**
```

Comment:
Please could you switch these around so external is first and internal is second. I don't know why eslint isn't catching these though 🤔 it shouldn't let you commit with the file like this.